crates/yas_core/src/contracts/yas_pool.cairo

```diff
@@ -1,6 +1,9 @@
 use starknet::ContractAddress;
 use yas_core::numbers::signed_integer::{i32::i32, i256::i256};
 use yas_core::numbers::fixed_point::implementations::impl_64x96::FixedType;
+use yas_core::contracts::yas_pool::YASPool::Slot0;
+use yas_core::libraries::tick::{Tick, Tick::TickImpl};
```

Comment:
```suggestion
use yas_core::libraries::tick::Info;
```

Reply:
can't, it will conflict will Position::Info used in same scope

Reply:
To avoid conflicts with another struct of the same name, you can use `as`. 
Example: 
```use yas_core::libraries::tick::Info as TickInfo;```